_use-case-gallery/concept-linking/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Smart Concept Linking
+permalink: /use-case-gallery/concept-linking/
+# Smart Concept Linking
+---
+
+This project serves as an example for using Interactive Concept Linking supported by 
+the recommendation functionalities of INCEpTION.
+
+There are five documents in this example project:
+* __NER-de-test.tsv.conll__: a well-annotated document from the [GermEval2014NER](https://sites.google.com/site/germeval2014ner/data) project, 
+converted into __conll__ format.
+* __Emmanuel_Macron_QA.txt__: an interview by WIRED with [Emmanuel Macron](https://corpling.uis.georgetown.edu/gum/index.html#annos)
+* __Michael_Jordan_scientist__: an excerpt of the Wikipedia article [Noam Chomsky](https://en.wikipedia.org/wiki/Noam_Chomsky)
+* __Tesla-Britannica__: an excerpt of the Britannica article [Nikola Tesla](https://www.britannica.com/biography/Nikola-Tesla)
+* __Wikipedia-Obama__: an excerpt of the Wikipedia article [Barack Obama](https://en.wikipedia.org/wiki/Barack_Obama)
+
+The following recommenders are configured in this project to give suggestions for named entity values and identifiers:
+
+* Token String Matcher - __value__
+* Token Sequence Classifier (OpenNLP NER) - __value__
+* Token String Matcher - __identifier__
+* Named Entity Linker - __identifier__
+
+___Setting up a Knowledge Base___
+Currently, Concept Linking only works with a Virtuoso KB.
+To set up Virtuoso, please clone the [Github](https://github.com/openlink/virtuoso-opensource) repository and follow the instructions.
+Further installation information can be found [here](https://www.openlinksw.com/blog/~kidehen/?id=1647).
+Wikidata offers canonical [RDF dumps](https://dumps.wikimedia.org/wikidatawiki/entities/) using the Turtle format. 
```

Comment:
Which of the dumps should be loaded? The "all" or the "truthy" ones? Did you check that these dumps are actually compatible with the hard-coded IRIs you are afaik still using in the entity linker code?